libs/application/templates/parental-leave/src/constants.ts

```diff
@@ -6,3 +6,9 @@ export enum API_MODULE_ACTIONS {
   assignEmployer = 'assignEmployer',
   sendApplication = 'sendApplication',
 }
+
+export enum StartDateOptions {
+  ESTIMATED_DATE_OF_BIRTH = 'estimatedDateOfBirth',
+  DATE_OF_BIRTH = 'dateOfBirth', // The actual date of birth
```

Comment:
I would rename it to make it explicit and to not need the comment

```suggestion
  ACTUAL_DATE_OF_BIRTH = 'actualDateOfBirth',
```

Reply:
Good point, should be updated now.